getting_started/eeglab.md

```diff
@@ -9,34 +9,66 @@ tags: [dataformat, eeglab, eeg]
+## Complementary use of both toolboxes
 
-## Reading data in EEGLAB .set format
+Processing data from EEGLAB through FieldTrip functions and vice-versa is made easy by the functions **[eeglab2fieldtrip](https://github.com/fieldtrip/fieldtrip/blob/master/external/eeglab/eeglab2fieldtrip.m)** and **[fieldtrip2eeglab](https://github.com/fieldtrip/fieldtrip/blob/master/external/eeglab/eeglab2fieldtrip.m)**.
+Here is an example code showing how to use those functions:
 
-In EEGLAB you can write your (pre)processed data to disk in the `.set` format, which are accompanied by an `.fdt` file. The `.set` file is actually a MATLAB file in disguise, you can read it using
+- For users working on EEGLAB, first save data on disk: File → Save current dataset (must be a `.set` file)
 
-   load('filename.set', '-mat')
-   
-which will give you an `EEG` structure in your main workspace. The accompanying `.fdt` files contain the time-series data of all EEG channels in a multiplexed float32 format.
+- (EEGLAB -> FieldTrip) Two methods can be used to import the `.set` file as a FieldTrip structure:
+	1. (recommended) Using **[ft_preprocessing](https://github.com/fieldtrip/fieldtrip/blob/release/ft_preprocessing.m)**:
+	```
+	cfg 		= [];
+	cfg.dataset = 'filename.set';
+	ft_data1	= ft_preprocessing(cfg);
+	```
+	2. Load the dataset through **[pop_loadset](https://sccn.ucsd.edu/~arno/eeglab/auto/pop_loadset.html)** and transform it to FieldTrip structure using **[eeglab2fieldtrip](https://github.com/fieldtrip/fieldtrip/blob/release/external/eeglab/eeglab2fieldtrip.m)** function:
+	```
+	EEG = pop_loadset(filename);
+	ft_data2 = eeglab2fieldtrip(EEG, 'raw');
+	```
 
-The EEGLAB `.set` and `.fdt` formats are directly supported by the low-level **[ft_read_header](https://github.com/fieldtrip/fieldtrip/blob/release/fileio/ft_read_header.m)**, **[ft_read_data](https://github.com/fieldtrip/fieldtrip/blob/release/fileio/ft_read_data.m)** and **[ft_read_event](https://github.com/fieldtrip/fieldtrip/blob/release/fileio/ft_read_event.m)** functions. This also means that you can use the high-level **[ft_definetrial](https://github.com/fieldtrip/fieldtrip/blob/release/ft_definetrial.m)** and **[ft_preprocessing](https://github.com/fieldtrip/fieldtrip/blob/release/ft_preprocessing.m)** functions for your analysis script, as explained in the [tutorials](/tutorial).
+- (FieldTrip -> EEGLAB) For data that has been processed in FieldTrip, it can be converted to EEGLAB format using **[fieldtrip2eeglab](https://github.com/fieldtrip/fieldtrip/blob/release/external/eeglab/eeglab2fieldtrip.m)** function. Note that the data (second argument) have to be passed as an array (not a cell):
+```
+eeglab_data = fieldtrip2eeglab(ft_data.hdr,cat(3,ft_data.trial{:}));
+pop_saveset(eeglab_data, 'filename', 'myfilename.set')
+```
+
+** THIS EXAMPLE CODE SHOULD BE CHANGED ACCORDING TO A NEW "fieldtrip2eeglab" FUNCTION THAT CONSIDERS FIELDTRIP DATA STRUCTURE **
```

Comment:
I suggest to leave out the comment here, and make an issue to change fieldtrip2eeglab in the appropriate repository. After the issue is resolved, we can update the documentation here, explaining that one can do either


    eeglab_data1 = fieldtrip2eeglab(hdr, dat, evt); where evt is optional

or 

    eeglab_data2 = fieldtrip2eeglab(data); % where data is a FT raw structure

Reply:
for reference, see https://github.com/fieldtrip/fieldtrip/issues/1857